web-frontend/src/pages/SelectAccount/SelectAccount.tsx

```diff
@@ -1,13 +1,13 @@
-import LandingPage from '../../layouts/LandingPage/LandingPage';
 import { FiUser, FiPlus } from 'react-icons/fi';
-import { Button } from '@massalabs/react-ui-kit/src/components/Button/Button';
-import { AccountSelector } from '@massalabs/react-ui-kit/src/components/AccountSelector/AccountSelector';
-import { MassaToken } from '@massalabs/react-ui-kit/src/components/Icons/Svg/SvgComponent/MassaToken';
-import { toMAS } from '@massalabs/massa-web3';
 import { Link, useNavigate } from 'react-router-dom';
 import { goToErrorPage, routeFor } from '../../utils';
 import useResource from '../../custom/api/useResource';
 import { AccountObject } from '../../models/AccountModel';
+import { toMAS } from '@massalabs/massa-web3';
+
+import LandingPage from '../../layouts/LandingPage/LandingPage';
+import { Button, AccountSelector } from '@massalabs/react-ui-kit';
+import { MassaToken } from '@massalabs/react-ui-kit/src/components/Icons/Svg/Massa/MassaToken';
```

Comment:
MassaToken is not exported in the root index?
`import { Button, AccountSelector, MassaToken } from '@massalabs/react-ui-kit';` would be nice

Reply:
No! He is not export in the `index.ts`.
But I can add it in.

Reply:
> MassaToken is not exported in the root index? `import { Button, AccountSelector, MassaToken } from '@massalabs/react-ui-kit';` would be nice


No! He is not export in the index.ts.
But I can add it in.

Reply:
Is there any reason to use the whole path!? We should be able to import it only with from '@massalabs/react-ui-kit'

Reply:
Are we exporting the SVG's on ui-kit!?

Reply:
No! We doesn't export SVG on UI-KIT. I will make a PR to solve this problem.

Reply:
@mazmassa Done one the ui-kit. So I will update package for kit when ui-kit is ready.

Reply:
Great! Thanks!

Reply:
Done on branch @peterjah